tests/testthat/test-forecasters-basics.R

```diff
@@ -1,5 +1,5 @@
 source(here::here("R", "load_all.R"))
-local_edition(3)
+testthat::local_edition(3)
```

Comment:
Isn't the default already 3? https://testthat.r-lib.org/reference/local_edition.html?q=local_edition#ref-usage

Reply:
For some reason it was thinking I should be using version 2; definitely going to need further work, the test fixes here are only partial